server/service/client_scripts.go

```diff
@@ -36,6 +37,15 @@ func (c *Client) RunHostScriptSync(hostID uint, scriptContents []byte) (*fleet.H
 			return nil, fmt.Errorf("decoding %s %s response: %w, body: %s", verb, path, err, b)
 		}
 	case http.StatusForbidden:
+		errMsg, err := extractServerErrMsg(verb, path, res)
```

Comment:
Left this in just in case we somehow get to calling the endpoint and scripts are disabled.